internal/jobs/transform.go

```
@@ -638,12 +648,30 @@ func (httpTransform *HTTPTransform) transformEntities(
 	// create headers if needed
 	url := httpTransform.URL
 
-	// set up our request
-	jsonEntities, err := json.Marshal(entities)
-	if err != nil {
-		return nil, err
+	var jsonData []byte
+	var err error
+	if httpTransform.SupportContext {
+		// add context to the request
+		ctx := httpTransform.NamespaceManager.GetContext(nil)
+
+		// create new array with ctx first
+		allEntities := make([]any, 0)
+		for _, entity := range entities {
+			allEntities = append(allEntities, entity)
+		}
+
+		// add context to the request at start of array
+		allEntities = append([]any{ctx}, allEntities...)
+
+		jsonData, _ = json.Marshal(allEntities)
```

we should probably not swallow the error, but log+return it